roles/confluent.kafka_connect_replicator/tasks/main.yml

```diff
@@ -0,0 +1,314 @@
+    kerberos_group: "{{kafka_connect_replicator_group}}"
+    kerberos_user: "{{kafka_connect_replicator_user}}"
+    kerberos_keytab_path: "{{kafka_connect_replicator_consumer_kerberos_keytab_path}}"
+    kerberos_keytab_destination_path: "{{kafka_connect_replicator_consumer_keytab_path}}"
+    kerberos_handler: "Restart Kafka Connect Replicator"
+  when: kafka_connect_replicator_consumer_kerberos_principal is defined
+
+- name: Configure Kerberos for Kafka Connect Replicator Producer
+  include_role:
+    name: confluent.kerberos
+  vars:
+    kerberos_group: "{{kafka_connect_replicator_group}}"
+    kerberos_user: "{{kafka_connect_replicator_user}}"
+    kerberos_keytab_path: "{{kafka_connect_replicator_producer_kerberos_keytab_path}}"
+    kerberos_keytab_destination_path: "{{kafka_connect_replicator_producer_keytab_path}}"
+    kerberos_handler: "Restart Kafka Connect Replicator"
+  when: kafka_connect_replicator_producer_kerberos_principal is defined
+
+- name: Configure Kerberos for Kafka Connect Replicator Monitoring Interceptors
+  include_role:
+    name: confluent.kerberos
+  vars:
+    kerberos_group: "{{kafka_connect_replicator_group}}"
+    kerberos_user: "{{kafka_connect_replicator_user}}"
+    kerberos_keytab_path: "{{kafka_connect_replicator_monitoring_interceptor_kerberos_keytab_path}}"
+    kerberos_keytab_destination_path: "{{kafka_connect_replicator_monitoring_interceptor_keytab_path}}"
+    kerberos_handler: "Restart Kafka Connect Replicator"
+  when: kafka_connect_replicator_monitoring_interceptor_kerberos_principal is defined
+
+- name: Create Kafka Connect Replicator Config directory
+  file:
+    path: "{{ kafka_connect_replicator.replication_config_file | dirname }}"
+    state: directory
+    mode: 0750
+    owner: "{{kafka_connect_replicator_user}}"
+    group: "{{kafka_connect_replicator_group}}"
+
+- name: Create Kafka Connect Replicator Log4j Config
+  template:
+    src: kafka-connect-replicator-log4j.properties.j2
+    dest: "{{kafka_connect_replicator.log4j_file}}"
+    mode: 0640
+    owner: "{{kafka_connect_replicator_user}}"
+    group: "{{kafka_connect_replicator_group}}"
+  when: kafka_connect_replicator_custom_log4j|bool
+  notify: Restart Kafka Connect Replicator
+
+- name: Create Kafka Connect Replicator Jolokia Config
+  template:
+    src: kafka-connect-replicator-jolokia.properties.j2
+    dest: "{{kafka_connect_replicator_jolokia_config}}"
+    mode: 0640
+    owner: "{{kafka_connect_replicator_user}}"
+    group: "{{kafka_connect_replicator_group}}"
+  when: kafka_connect_replicator_jolokia_enabled|bool
+  notify: Restart Kafka Connect Replicator
+
+- name: Deploy Kafka Connect Replicator JMX Exporter Config File
+  copy:
+    src: "kafka_connect_replicator.yml"
+    dest: "{{kafka_connect_replicator_jmxexporter_config_path}}"
+    mode: 0640
+    owner: "{{kafka_connect_replicator_user}}"
+    group: "{{kafka_connect_replicator_group}}"
+  when: kafka_connect_replicator_jmxexporter_enabled|bool
+
```

Comment:
Should add in our backlog to investigate secrets protection because connect works w secrets protection, shouldn't replicator also if it is just running connect

Reply:
I've already investigated this!!!! I've spoken with the replicator team ,it's not supported with the replicator binary!  Can you please review the code and stop ripping apart a design you already had a change to review in the one pager.